grpc-protocol/src/main/java/com/linecorp/armeria/server/grpc/protocol/AbstractUnsafeUnaryGrpcService.java

```diff
@@ -72,11 +88,34 @@
      */
     protected abstract CompletionStage<ByteBuf> handleMessage(ServiceRequestContext ctx, ByteBuf message);
 
+    @Nullable
+    private static SerializationFormat resolveSerializationFormat(HttpRequest req) {
+        @Nullable
+        final MediaType contentType = req.contentType();
```

Comment:
nit: Conventionally, we don't use nullable for a local variable.
```suggestion
        final MediaType contentType = req.contentType();
```

Reply:
I'm curious if we need to change our policy now that IntelliJ IDEA started to complain about nullability of local variables. :thinking:

Reply:
Done. Quick question though: I'm using inspections config suggested in https://armeria.dev/community/developer-guide#setting-up-your-ide and it seems to be unhappy about nullable local variables without annotation, should I just ignore it?

![Screen Shot 2021-07-26 at 7 35 07 PM](https://user-images.githubusercontent.com/86681657/127086218-75e1c7ed-a946-49c9-ba5c-60e0a0784318.png)

Reply:
> and it seems to be unhappy about nullable local variables without annotation, should I just ignore it?

You may want to change the default `@Nullable` annotation. 
<img width="2038" alt="image" src="https://user-images.githubusercontent.com/1866157/127086657-15c5c25b-1fad-4557-a3e4-ff86e6082aee.png">

Reply:
> I'm curious if we need to change our policy now that IntelliJ IDEA started to complain about nullability of local variables.

That sounds good to me. It would help null-checking for local variables though it is a bit verbose.

Reply:
@ikhoon should I revert my last commit? :)

Reply:
You don't need it at the moment. :-) We can apply `@Nullable` for local variables later once we arrive at a decision.